### DISCUSSION TITLE
Show MSI Dialog

### DISCUSSION THREAD

**Discussion by ad-chuliang** [wixtoolset/issues#8677] [Questions] [answered]
I was operating too fast and running a lot of programs at once, causing the system to get stuck. Then manually deleted files from the c-drive as well as the contents of the registry, but can't remember what I deleted. It all happened too fast and too chaotic. Now the bootloader won't enter the msi interface and won't install the features in the msi. Running the bootloader on other computers works fine. What could be the reason for this?

**Comment by bevanweiss** [comment, marked as answer]
The Windows Installer system might think that the product is installed, and so it wants to launch the modify/uninstall option instead of doing a new install, but you've corrupted the 'where to find' information for the windows installer to access the currently installed MSI.
It's why it's best to test installer packages in isolated environments (like VMs or the sandbox) where you can just roll them back to a known good state afterwards.

You could try using some of the 3rd party apps out there that analyse and 'repair' the add/remove programs settings.  There's just a lot of malware out there advertising the same, so be careful what you choose.

**Reply by ad-chuliang** [reply]
thanks.
